src/Command/RemoveDuplicateAssetCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace TorqIT\DuplicateAssetCleanupBundle\Command;
+
+use Pimcore\Console\AbstractCommand;
+use Pimcore\Db;
+use Pimcore\Model\Asset;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class RemoveDuplicateAssetCommand extends AbstractCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('torq:cleanup:most-duplicated-asset')
+            ->setDescription('Removes all duplicates for whichever asset has the most duplicates and updates all references' .
+        ' to that asset to point to the new unified asset.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        // TODO query to see how many duplicate files before asking?
+        $helper = $this->getHelper('question');
+        $question = new ConfirmationQuestion('This command cannot be undone. Are you sure you want to continue? [Yes | No]: ', false, '/^Yes|Y/i');
+
+        if (!$helper->ask($input, $output, $question)) {
+            $output->writeln("Command cancelled");
+            return 0;
+        }
+
+        $result = $this->getHashWithMostDuplicates();
+        $duplicates = $this->getDuplicateAssetsForHash($result["binaryFileHash"]);
+
+        $dupString = implode(", ", $duplicates);
+
+        //This message is temporary and is just here to demonstrate that the query is working
+        $this->output->writeln("Found asset with {$result["total"]} duplicates ($dupString)");
+
+        return 0;
+    }
+
+    private function getHashWithMostDuplicates()
+    {
+        return Db::get()->createQueryBuilder()
+            ->select("groupedVersions.binaryFileHash", "COUNT(1) AS total")
+            ->from("versions", "groupedVersions")
+            ->innerJoin("groupedVersions", "({$this->buildMostRecenVersionSubquery()})", "maxVersion", 
+                "groupedVersions.cid = maxVersion.cid AND groupedVersions.versionCount = maxVersion.version")
+            ->groupBy("groupedVersions.binaryFileHash")
+            ->orderBy("total", "DESC")
+            ->setMaxResults(1)
+            ->execute()
+            ->fetchAssociative();
```

Comment:
Although this is cool, I also am slightly confused how we'd test this against the real database to make sure it's working as expected. Can we breakpoint it to find the rendered SQL and then test that?